tests/aem_hw.pm

```diff
@@ -350,18 +452,31 @@ sub setup_aem {
     # cleanup in case AEM was previously initialized (useful for debugging this test)
     assert_script_run('rm -rf /var/lib/anti-evil-maid/aem');
 
-    assert_script_run("grub2-install $boot_disk");
+    # no need to install grub on EFI installations
```

Comment:
Not quite, it still needs to be installed, but it is done automatically by RPM hooks for EFI, so maybe something like:
```suggestion
    # no need to manually install grub on EFI installations
```